variables.tf

```
@@ -27,6 +27,24 @@ variable "log_analytics_workspace_id" {
   description = "The ID of the Log Analytics workspace to send diagnostics to."
   type        = string
 }
+variable "log_analytics_destination_type" {
+  description = "The type of log analytics destination to use for this Log Analytics Workspace."
+  type        = string
+  default     = null
+}
+
+variable "diagnostic_setting_name" {
+  description = "The name of this diagnostic setting."
+  type        = string
+  default     = "audit-logs"
```

Make the default value for the `diagnostic_setting_name` variable the same as the value previously hardcoded in `main.tf`, to avoid breaking changes.

```suggestion
  default     = "failure-logs"
```